docs/List.md

```diff
@@ -642,6 +642,8 @@ export const PostList = () => (
 );
 ```
 
+**Note**: If you've added a [`Pagination`](#pagination-pagination-component) component to your `List` component, make sure to set `perPage` to a value that exists in `rowsPerPageOptions`
```

Comment:
I'd put it differently: I'd explain that the default pagination includes per page options of 5, 10, and 25. If the developer sets the `perPage` to a value not in that set, they must also customize the pagination so that it allows this value, or else there will be an error. And you could show how to do that.

By the wayn I think we should add 50 as a possible perPage option.